gui/src-tauri/src/main.rs

```diff
@@ -103,6 +127,11 @@ fn main() {
 		// We don't do anything with the job anymore, but we shouldn't drop it because that would
 		// terminate our process tree. So we intentionally leak it instead.
 		std::mem::forget(job);
+
+		if !webview2_exists() {
+			show_error("Couldn't find WebView 2 installed. You can install it from Microsoft's site or try reinstalling it from the SlimeVR installer");
```

Comment:
Maybe link to Microsoft's site for WebView2 and link to the installer, this doesn't guide users very well (Maybe just say to use the installer if we just want it simple for users? They might get confused with 2 "random" links)

Reply:
I cant have stuff like hyperlinks with names, I would be adding a whole URL